mods/cnc/languages/lua/en.ftl

```diff
@@ -101,12 +101,12 @@ steal-nuclear-detonator = Steal the GDI nuclear detonator.
 move-to-evacuation-point = Move to the evacuation point.
 
 ## nod06ab
-destroy-gdi-supporter-houses = Destroy the houses of the GDI supporters
+destroy-gdi-supporter-houses = Destroy the houses of GDI supporters
```

Comment:
The "the" should probably stay? This will be referring to a specific group of GDI supporters, whereas the changed wording implies anyone.